ports/unix/variants/coverage/mpconfigvariant.mk

```diff
@@ -9,9 +9,12 @@ CFLAGS += \
 	-Wold-style-definition -Wpointer-arith -Wshadow -Wuninitialized -Wunused-parameter \
 	-DMICROPY_UNIX_COVERAGE
 
+CFLAGS_EXTRA = -DMODULE_CEXAMPLE_ENABLED=1 -DMODULE_CPPEXAMPLE_ENABLED=1
```

Comment:
I think this should be `CFLAGS += ...`, so that `CFLAGS_EXTRA` can still be set on the command line to build this variant with additional options

Reply:
ok